docs/pbc/all/multi-factor-authentication/202505.0/install-multi-factor-authentication-feature.md

```diff
@@ -38,11 +42,22 @@ Make sure the following modules have been installed:
 
 Set up the following configuration.
 
+{% info_block infoBox "Customer and User configuration separation" %}
```

Comment:
By user do you mean BO users?

Reply:
From a database perspective, Agent users, Backoffice users (and soon Merchant Portal users) are all the same user entity. And that’s exactly what this refers to.